### ISSUE TITLE
Handling additional tif files in the input directory

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-tasks-core#214]
This is a bug with a current workaround. So not urgent to address, but good to pick up when we update any filename parsing.

----

As part of the filename refactor (see https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/189), there is an extra edge-case we missed:

Depending on the microscope type, there is some metadata in the input folder. For example, the Yokogawa creates a few camera specification files. Those are named something like:
```
DC_sCMOS #1_CAM1.tif
DC_sCMOS #2_CAM2.tif
SC_BP445-45_20x_M01_CH01.tif
SC_BP525-50_20x_M01_CH02.tif
SC_BP600-37_20x_M01_CH03.tif
SC_BP676-29_20x_M01_CH04.tif
```

(with slight variations depending on camera, objectives & laser paths being used).
These images do not need to be parsed into the OME-Zarr. For the time being, they can just be ignored. This didn't really come up in the synthetic test cases and the test cases that contained PNGs. But if we have TIF images (directly from the microscope) and select them with the glob "*.tif", then the parsing now fails for them:

```
2022-11-21 12:48:34,430; INFO; START create_zarr_structure task
Traceback (most recent call last):
  File "/net/nfs4/pelkmanslab-fileserver-common/data/homes/fractal/joel/fractal_v1/fractal-demos/examples-v1/server/FRACTAL_ROOT/.fractal/fractal-tasks-core0.4.0/venv/lib/python3.8/site-packages/fractal_tasks_core/create_zarr_structure.py", line 386, in <module>
    run_fractal_task(
  File "/data/homes/fractal/joel/fractal_v1/fractal-demos/examples-v1/server/FRACTAL_ROOT/.fractal/fractal-tasks-core0.4.0/venv/lib/python3.8/site-packages/fractal_tasks_core/_utils.py", line 89, in run_fractal_task
    metadata_update = task_function(**task_args.dict())
  File "/net/nfs4/pelkmanslab-fileserver-common/data/homes/fractal/joel/fractal_v1/fractal-demos/examples-v1/server/FRACTAL_ROOT/.fractal/fractal-tasks-core0.4.0/venv/lib/python3.8/site-packages/fractal_tasks_core/create_zarr_structure.py", line 113, in create_zarr_structure
    filename_metadata = parse_filename(fn.name)
  File "/data/homes/fractal/joel/fractal_v1/fractal-demos/examples-v1/server/FRACTAL_ROOT/.fractal/fractal-tasks-core0.4.0/venv/lib/python3.8/site-packages/fractal_tasks_core/lib_parse_filename_metadata.py", line 98, in parse_filename
    raise ValueError(f"Something wrong with {filename=}, {TFLAZC=}")
ValueError: Something wrong with filename='SC_BP445-45_20x_M01_CH01', TFLAZC='CH01'
```

Those files should be ignored. We could hard-code it for the Yokogawa by providing an exception list, e.g.:
Ignore files starting with `DC_sCMOS`or `SC_`.
A nice implementation could be: If we run into the ValueError for the parsing like above, but files fit one of the patterns, ignore them.

Not yet sure how this will look on other microscopes.

-----

Workaround: Move the DC & SC files to a separate folder
Or create a link to the image files & MeasurementData/Details files to a new folder (`ln old_folder/BEGINNING_FILENAME* new_folder`)

**Comment by tcompa** [comment]
First solution: let's switch back to the previous behavior (for any invalid name, print a log and skip it).
Later on we can add something like an `ignore_pattern` variable.

**Comment by tcompa** [comment]
Fixed in 0.4.6 (https://fractal-tasks-core.readthedocs.io/en/stable/changelog.html).
